api_calls/reference/contacts/update-request.json

```diff
@@ -0,0 +1,10 @@
+{
+  "special_offers": [
```

Comment:
@radekzawada special offers in contacts json?:P

Reply:
haha, indeed doesn't match, fixed